indico/modules/auth/blueprint.py

```diff
@@ -34,3 +36,14 @@
 _bp.add_url_rule('/register/<provider>', 'register', RHRegister, methods=('GET', 'POST'))
 
 _bp.add_url_rule('/reset-password/', 'resetpass', RHResetPassword, methods=('GET', 'POST'))
+
+
+with _bp.add_prefixed_rules('/user/<int:user_id>', '/user'):
+    _bp.add_url_rule('/accounts/', 'accounts', RHUserAccounts, methods=('GET', 'POST'))
+    _bp.add_url_rule('/accounts/remove/<identity>', 'remove-account', RHUserAccountsRemove)
```

Comment:
:+1: for POST and the composition of the url.
I'm not so sure about using 'identity' everywhere, specially in places more closely related with the view... But if we go for that, I would change also the RH name. Thoughts on this?